pom.xml

```diff
@@ -116,6 +116,11 @@
             <version>3.0.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Hi @tjaehnel, I'd like to move this importer to a separate repository, since we want to have no runtime dependencies except the Java SDK for the OpenFastTrace core. @kaklakariada created the service loader so that importers and exporters can be added at program start from regular JAR files. Once we switch to Java 11 (#206), this will be even more comfortable. Basically the importer JAR simply needs to be present in the class path at start-up.

Reply:
@redcatbear can you give me a hint how to register with the service loader and where to put the jar file until transition to Java 11 is done?

Reply:
In your JAR archive add a property file called `org.itsallcode.openfasttrace.exporter.ExporterFactory` as resource. The content of all files of this name is automatically merged. The JAR itself needs to be in the class path at start time.